syft/frameworks/torch/pointers/pointer_tensor.py

```diff
@@ -136,7 +136,12 @@ def data(self, new_data):
         self._data = new_data
 
     def is_none(self):
-        return self.owner.request_is_remote_tensor_none(self)
+        try:
+            return self.owner.request_is_remote_tensor_none(self)
+        except:
```

Comment:
Could we also just throw this in a log or something?

Reply:
meaning like logging.Debug?

Reply:
is logging already setup?

Reply:
Maybe not. Is there an issue for this todo? If so maybe just make it high priority

Reply:
How about I make a META TODO - which is for someone to search the codebase for all the TODOs and make issues for them

Reply:
Lol. That would be perfect

Reply:
Also for extra credit if they could tag the issue in each of the todos that would be amazing

Reply:
I'm on it.

Reply:
btw, maybe we want to install this: https://todo.jasonet.co/ (it will do the job for future TODOs :smile: )

Reply:
Big Fan of this

Reply:
Done! please let's use https://todo.jasonet.co/  from now on :weary:

Reply:
@mari-linhares would it make sense to prioritize these todos? looks like many of these :) Shall we use Github project to sort these?

Reply:
Honestly, a lot of them are not totally clear... so I'm not sure about prioritizing them. But certainly, we want to "get rid" of them. We should tag the easy todo fixes as "good-first-issues".